apps/daimo-mobile/src/storage/storedTypes.ts

```diff
@@ -241,12 +241,14 @@ interface StoredV16ForeignCoin {
 
 export interface StoredV15LandlineAccount {
   daimoAddress: Address;
+  landlineAccountUuid: string;
   bankName: string;
+  bankLogo: string | null;
   accountName: string;
-  lastFour: string;
+  accountNumberLastFour: string;
+  bankCurrency: string;
   liquidationAddress: Address;
-  chain: string;
-  destinationCurrency: string;
-  bankLogo?: string;
+  liquidationChain: string;
+  liquidationCurrency: string;
```

Comment:
hmm, adding non-optional fields to an existing StoredModel is generally bad & can cause crashes.

It should be OK in this case since we haven't shipped Landline > nobody except us should have these in their stored account.

Can you confirm that nobody outside Daimo could have one of these yet?

Reply:
the landline whitelist was empty before, so there shouldn't be anyone with `StoredV15LandlineAccount` populated